.github/workflows/deploy_to_aws.yml

```
@@ -156,8 +156,28 @@ jobs:
+        with:
+          string: ${{ vars.PROJECT }}
+
       - name: Move static
-        run: mv src/react/build src/django/static
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          FRONTEND_BUCKET: ${{ steps.project.outputs.lowercase }}-${{ steps.get_env_name.outputs.lowercase }}-frontend
+          AWS_DEFAULT_REGION: "eu-west-1"
+          CLOUDFRONT_DOMAIN: ${{ vars.CLOUDFRONT_DOMAIN }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:59: Double quote to prevent globbing and word splitting

(shellcheck)